test/unit/utils/check.test.ts

```diff
@@ -0,0 +1,22 @@
+const { expect } = require('chai');
+const { isColString } = require('sequelize/lib/utils/check');
```

Comment:
We could use `import` here since it's a typescript file

Reply:
I tried and it didn't work. let me check

Reply:
like this?

```import { expect } from "chai";
import { Utils } from "sequelize";

const { isPrimitive, isColString, canTreatArrayAsAnd, defaultValueSchemable } =
  Utils;
```

Reply:
Should work, but I expected this to work too:
```
import { isColString } from 'sequelize/lib/utils/check';
```

Reply:
done

Reply:
aha! somehow didn't work before but here we go